Client/mods/deathmatch/logic/CClientManager.cpp

```diff
@@ -55,6 +55,7 @@ CClientManager::CClientManager()
     m_pPacketRecorder = new CClientPacketRecorder(this);
     m_pImgManager = new CClientIMGManager(this);
     m_pBuildingManager = new CClientBuildingManager(this);
+    m_p2DFXManager = new CClient2DFXManager(this);
```

Comment:
Maybe unique_ptr?

Reply:
Not in this case, to maintain consistency with the rest of the class